src/main/java/hudson/remoting/Launcher.java

```diff
@@ -144,9 +144,10 @@ public void setTextMode(boolean b) {
     public String proxyCredentials = null;
 
     @Option(name="-cp",aliases="-classpath",metaVar="PATH",
-            usage="add the given classpath elements to the system classloader.")
+            usage="add the given classpath elements to the system classloader. (DEPRECATED)")
```

Comment:
Should this `docker-inbound-agent` launch script be updated to not use `-cp`? https://github.com/jenkinsci/docker-inbound-agent/blob/master/jenkins-agent#L121

Reply:
No, that is the `--classpath` / `-cp` option to the `java` launcher. This PR is about an application option, like

```bash
java -jar agent.jar -cp …
```